app/services/return-logs/setup/submit-check.service.js

```diff
@@ -27,6 +29,35 @@ const SessionModel = require('../../../models/session.model.js')
  */
 async function go(sessionId, user) {
   const session = await SessionModel.query().findById(sessionId)
+
+  const validationResult = _validate(session)
+
+  if (!validationResult) {
```

Comment:
The convention now is to just name the result `error`.


```suggestion
  const error = _validate(session)

  if (!error) {
```

Reply:
New convention used